Sources/ReleaseServer/ReleaseServer.WebApi.Test/Mappers/DeploymentMetaInfoMapperTest.cs

```diff
@@ -1,30 +1,48 @@
 using System;
 using System.IO;
 using FluentAssertions;
-using ReleaseServer.WebApi.Config;
 using ReleaseServer.WebApi.Mappers;
+using ReleaseServer.WebApi.Models;
+using ReleaseServer.WebApi.Test.Utils;
 using Xunit;
 
 namespace ReleaseServer.WebApi.Test
 {
     public class DeploymentMetaMapperTest
     {
+        private readonly string ProjectDirectory;
+        private readonly DeploymentMetaInfo ExpectedMeta;
 
-        [Fact]
-        public void ConvertJsonToDeploymentMetaInfo()
+        public DeploymentMetaMapperTest()
         {
-            var projectDirectory = Directory.GetParent(Environment.CurrentDirectory).Parent.Parent.FullName;
-
-            var expectedMeta = new DeploymentMetaInfo
+            ProjectDirectory = TestUtils.GetProjectDirectory();
+            ExpectedMeta =  new DeploymentMetaInfo
             {
                 ChangelogFileName = "changelog.txt",
                 ArtifactFileName = "artifact.zip",
                 ReleaseDate = new DateTime(2020, 02, 01)
             };
+        }
+        
+
+        [Fact]
+        public void ConvertJsonToDeploymentMetaInfo_String()
+        {
+            var parsedMeta = DeploymentMetaInfoMapper.ParseDeploymentMetaInfo(Path.Combine(ProjectDirectory, "TestData","testDeployment.json"));
+
+            parsedMeta.Should().BeEquivalentTo(ExpectedMeta);
+        }
+        
+        [Fact]
+        public void ConvertJsonToDeploymentMetaInfo_ByteArray()
+        {
+            var testMetaInfoByteArray = File.ReadAllBytes(Path.Combine(ProjectDirectory, "TestData", "testDeployment.json"));
+            
+            var parsedMeta = DeploymentMetaInfoMapper.ParseDeploymentMetaInfo(testMetaInfoByteArray);
 
-            var parsedMeta = DeploymentMetaInfoMapper.ParseDeploymentMetaInfo(Path.Combine(projectDirectory, "TestData","testDeployment.json"));
+            string test = null;
```

Comment:
This variable is unused.

Reply:
This test got removed in the second PR. 
I will remove it anyways:)

Reply:
Done, see https://github.com/Traeger-GmbH/release-server/pull/58/commits/f79a7b0384bf265984b2328cf1923beea7e9d023